test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -17,8 +17,8 @@ public void setup() throws IllegalValueException {
     }
     
     @Test
-    public void test() {
-        fail("Not yet implemented");
+    public void test_isSimilar_true() {
```

Comment:
Wrong method name, according to the coding convention it should be like this `isSimilar_nullParams_returnTrue`. Start from the method you want to test, then what you want to test and finally the expected return value.